backend/drive/drive.go

```diff
@@ -144,6 +146,8 @@ func driveScopes(scopesString string) (scopes []string) {
 		scope = strings.TrimSpace(scope)
 		scopes = append(scopes, scopePrefix+scope)
 	}
+	// The minimal read-only scope for activities should be sufficient
+	scopes = append(scopes, scopePrefix+"drive.activity.readonly")
```

Comment:
Shouldn't this go in `defaultScope`?

Reply:
Problem with that is that any scope override from the config causes the `defaultScope` to not be used.